### PR TITLE
adding test for en locales for removing privacy policy

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

File: components/single-form.jsx [hunk 1]
```diff
@@ -145,7 +156,14 @@ module.exports = React.createClass({
     }
     this.paypal(validate, submit);
   },
+  checkPrivacyPolicyTest: function() {
+    return this.props.privacyPolicyTest && this.props.locales[0].indexOf("en") === 0;
```
Comment (jbuck) [inline components/single-form.jsx]: Lets not do a check for locale here

File: pages/one-page-policy-a.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    var className = "row new-flow-test";
+    if (this.props.test) {
+      className += " " + this.props.test;
+    }
+    return (
+      <div className={className}>
+        <Header locale={this.props.locales[0]} alt={this.getIntlMessage('donate_to_mozilla')}></Header>
+        <SingleForm
+          billingAddress={true}
+          privacyPolicyTest="We care about privacy and about helping you make informed choices. We want you to understand that your payment details will be processed by Stripe (for credit/debit cards) or Paypal, and a record of your donation will be stored by Mozilla."
```
Comment (jbuck) [inline pages/one-page-policy-a.jsx]: Can you make this the key used for Intl? Maybe like on line 16?

**Comment by jbuck** [comment]
Could you add the privacy policy variants to messages.properties?

**Comment by jbuck** [comment]
Fix #1167

**Comment by ScottDowne** [comment]
@jbuck fixed.

File: locales/en-US/messages.properties [hunk 3]
```diff
@@ -123,3 +123,7 @@ red_heart_the_web={redHeart} the web?
 h1_popup_further_monlth=Before we process your donation, will you consider stretching your contribution by converting it to a monthly gift of {amount}?
 popup_answer_yes=<b>Yes!</b> — Convert my one-time gift to an automatic monthly donation of {newAmount}
 popup_answer_no=<b>No thanks</b> — please continue with my one-time gift of {amount}
+
+# privacy policy varients
+privacy_policy_a=We care about privacy and about helping you make informed choices. We want you to understand that your payment details will be processed by Stripe (for credit/debit cards) or Paypal, and a record of your donation will be stored by Mozilla.
```
Comment (jbuck) [inline locales/en-US/messages.properties]: Nit: could you change this to privacy_policy_var_a?

**Comment by jbuck** [comment]
@ScottDowne r+ with that one nit fixed (for both variants)

**Pull request merged**